django/contrib/auth/views.py

```diff
@@ -150,8 +155,9 @@ def get_next_page(self):
                 self.redirect_field_name,
                 self.request.GET.get(self.redirect_field_name)
             )
-            # Security check -- don't allow redirection to a different host.
-            if not is_safe_url(url=next_page, host=self.request.get_host()):
+            # Security check -- Ensure the user-originating redirection URL is
+            # safe.
+            if not is_safe_url(url=next_page, host=self.get_success_url_allowed_hosts()):
```

Comment:
Do you think it could be worth deprecating the `host` parameter of `is_safe_url` in favor of an `allowed_hosts` one now that it accepts multiple values?

This function is not part of the public API but a lot of third-party applications rely on it.